static/app/components/globalDrawer/components.tsx

```diff
@@ -58,14 +58,20 @@ export function DrawerPanel({
 }: DrawerPanelProps & {
   ref?: React.Ref<HTMLDivElement>;
 }) {
-  const {panelRef, resizeHandleRef, handleResizeStart, persistedWidthPercent} =
-    useDrawerResizing({
-      drawerKey,
-      drawerWidth,
-    });
+  const {
+    panelRef,
+    resizeHandleRef,
+    handleResizeStart,
+    persistedWidthPercent,
+    isSmallScreen,
```

Comment:
this feels like it shouldn't come back from the hook actually, it's sort of unrelated to the logic of doing drawer resizing